src/Blankslate/index.tsx

```diff
@@ -1 +1 @@
-export {default as Blankslate} from './Blankslate'
+export {default, BlankslateProps} from './Blankslate'
```

Comment:
Can we please keep `default as Blankslate` as is because the imports in the repo itself is relaying on the named exports. ([example](https://github.com/primer/react/blob/main/src/DataTable/DataTable.features.stories.tsx#L15))

```
export {default as Blankslate, BlankslateProps} from './Blankslate'
```